frontend/dashboard/relay.config.json

```diff
@@ -1,11 +1,11 @@
 {
   "src": "./src",
   "language": "typescript",
-  "schema": "./supergraph.graphql",
+  "schema": "./src/newworkflows.graphql",
```

Comment:
schema should remain at `supergraph.grahql` which will be added in by the CI

Reply:
I've put this back to supergraph